lodash missing for install script

The install script wants to make use of lodash.

I suspect that because I have a direct dependency on lodash, npm is deciding to install lodash at the top level only, and is also not finished doing so by the time my application's npm process tries to install sauce-connect-launcher.

All of this is to say that I get:

```
> sauce-connect-launcher@0.7.0 install /Users/ssalsbur/Documents/_ws/marklogic-samplestack/browser/node_modules/sauce-connect-launcher
> node scripts/install.js

|
module.js:340
    throw err;
          ^
Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/ssalsbur/Documents/_ws/marklogic-samplestack/browser/node_modules/sauce-connect-launcher/lib/sauce-connect-launcher.js:6:7)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
npm ERR! sauce-connect-launcher@0.7.0 install: `node scripts/install.js`
npm ERR! Exit status 8
```

(and I think it's because of npm timing).

I believe this might be solved by changing the install step to a prepublish step as per https://www.npmjs.org/doc/misc/npm-scripts.html .

I was going to try to such a change and then submit a PR if it helped, but the next problem I ran into is that when I try to install sauce-connect-launcher from the git repo, I get an error that grunt-simple-mocha could not be found in the registry, so I abandoned the concept of testing whether or not that change would help.

Do you think it would help to switch from a a `scripts: { install: "..." }` script to a `scripts: { prepublish: "..." }` script?

I saw this after updating my reference to sauce-connect-launcher from 0.6.1 to 0.7.0. I'm going to try reverting to see if that helps.

UPDATE: yes, reverting to and hardcoding 0.6.1 fixes the install issue.

--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     "url": "https://github.com/bermi/sauce-connect-launcher"
   },
   "scripts": {
-    "install": "node scripts/install.js",
+    "prepublish": "node scripts/install.js",
     "test": "make test"
   },
   "dependencies": {

